Install.md

```diff
@@ -87,19 +68,25 @@ This guide helps you build and install Snort.
     ```sh
     cd snort-2.9.8.3/
     ```
-    
-2. Run the configuration script.
+2. Prepare for automake, then autocreate makefile. 
     ```sh
-    ./configure --enable-sourcefire
+    aclocal
+    autoconf
+    autoheader
+    automake -a
+    ```
+3. Run the configuration script.
```

Comment:
Running into issues on this step:
<img width="408" alt="Screen Shot 2019-10-24 at 10 52 06 PM" src="https://user-images.githubusercontent.com/32916582/67539732-1399c680-f6b1-11e9-9b27-f57fd2589140.png">

Reply:
I think the issue is that the $PATH variable needs to be initialized to point to the `daq-modules-config` file. This file is found in `/daq-2.0.6/os-daq-modules`